app/enterprise/2.1.x/plugins/grpc.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Introduction to Kong gRPC Plugins
+---
+
+Before we dive into the specifics of how to configure Kong's gRPC plugins, let's
+first discuss the advantages of the protocol. Unlike JSON, [gRPC](https://en.wikipedia.org/wiki/GRPC)
+is a binary protocol, using [protobuf](https://en.wikipedia.org/wiki/Protocol_Buffers)
+definitions to instruct how the data is marshalled and unmarshalled. Because
+binary data is used instead of text, it's a more efficient way to transmit data
+over a network. But this also makes gRPC harder to work with because inspecting
+what went wrong is more challenging. Additionally, few clients natively handle gRPC.
```

Comment:
```suggestion
over a network. However, this also makes gRPC harder to work with, because inspecting
what went wrong is more challenging. Additionally, few clients natively handle gRPC.
```